crates/sui/src/keytool.rs

```diff
@@ -42,12 +46,16 @@ use sui_types::zk_login_authenticator::ZkLoginAuthenticator;
 use sui_types::zk_login_util::AddressParams;
 use tracing::info;
 
+use tabled::builder::Builder;
+use tabled::settings::Rotate;
+use tabled::settings::{object::Rows, Modify, Width};
+
 #[cfg(test)]
 #[path = "unit_tests/keytool_tests.rs"]
 mod keytool_tests;
 
 #[allow(clippy::large_enum_variant)]
-#[derive(Subcommand)]
+#[derive(Subcommand, Serialize)]
```

Comment:
Good catch, probably it doesn't. The `derivation_path` fields of some structs were using ` #[serde(skip_serializing)]`, so that's why it was required. I removed it for now and we'll see if anyone complains.